daemon/container_unix.go

```diff
@@ -1219,7 +1219,7 @@ func (container *Container) removeMountPoints(rm bool) error {
 			// not an error, but an implementation detail.
 			// This prevents docker from logging "ERROR: Volume in use"
 			// where there is another container using the volume.
-			if err != nil && err != store.ErrVolumeInUse {
+			if err != nil && strings.Contains(err.Error(), store.ErrVolumeInUse.Error()) {
```

Comment:
I wonder if we should have a `volume.IsErrVolumeInUse(err)` function, I've seen this check in a few places.